packages/@okta/vuepress-site/docs/reference/api/authn/index.md

```diff
@@ -90,16 +90,19 @@ The authentication transaction [state machine](#transaction-state) can be modifi
 
 The context object allows [trusted web applications](#trusted-application) such as an external portal to pass additional context for the authentication or recovery transaction.
 
-> **Note:** Overriding context such as `deviceToken` is a highly privileged operation limited to trusted web applications and requires making authentication or recovery requests with a valid administrator API token. If an API token isn't provided, the `deviceToken` is ignored.
-
-> **Caution:** The `deviceToken` parameter isn't shared between the Authentication API and the Okta Identity Engine-specific APIs. See [Upgrade to Okta Identity Engine](https://developer.okta.com/docs/guides/oie-upgrade-overview/main/).
-
-
 | Property    | Description                                                                   | DataType | Nullable | Unique | Readonly | MaxLength |
 | ----------- | ----------------------------------------------------------------------------- | -------- | -------- | ------ | -------- | --------- |
-| deviceToken | A globally unique ID identifying the user's client device or user agent | String   | TRUE     | FALSE  | FALSE    | 32          |
+| deviceToken | A globally unique ID (without hyphens) identifying the user's client device or user agent | String   | TRUE     | FALSE  | FALSE    | 32          |
+
+> **Caution:** The `deviceToken` parameter isn't shared between the Authentication API and the Okta Identity Engine-specific APIs. See [Upgrade to Okta Identity Engine](https://developer.okta.com/docs/guides/oie-upgrade-overview/main/).> **Caution:** The `deviceToken` parameter isn't shared between the Authentication API and the Okta Identity Engine-specific APIs. See [Upgrade to Okta Identity Engine](https://developer.okta.com/docs/guides/oie-upgrade-overview/main/).
 
-> **Note:** Always pass the same `deviceToken` for a user's device with every authentication request for per-device or per-session sign-on policy Factor challenges. If the `deviceToken` is absent or doesn’t match the previous `deviceToken`, the user is challenged every-time instead of per-device or per-session.<br><br>Similarly, you must always pass the same `deviceToken` for a user's device with every authentication request for **new device security Behavior Detection**. If the `deviceToken` is absent or doesn't match a recent `deviceToken` for the user, the request is considered to be from a new device. See [New Device Behavior Detection](https://help.okta.com/okta_help.htm?id=ext_proc_security_behavior_detection).
+> **Note:**
+>
+> * Overriding context, such as `deviceToken`, is a highly privileged operation limited to trusted web applications. This requires the application to make authentication or recovery requests with a valid administrator API token. If an API token isn't provided, the `deviceToken` is ignored.
```

Comment:
Could use 'using' instead of 'with', but 'with' is fine

Reply:
Slightly reworded.